todo-api/index.js

```diff
@@ -0,0 +1,33 @@
+var express = require("express");
```

Comment:
an interesting take to stand up a middleware server.
would it have been possible to expand this idea and have express also serve up the Vue app? Would there be any benefit to having express serve the Vue app alongside the middleware API? Any potential pitfalls?